### PR TITLE
[NP-52] Series Linking

### PR DESCRIPTION
**Pull request by legend620**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Review by ryan00012 — APPROVED** [review]

File: platform/core/src/services/ViewerToolsetService/ViewerToolset.js [hunk 1]
```diff
@@ -4,16 +4,16 @@ const publicAPI = {
   name,
   getState: _getState,
   setIsReferenceLinesEnabled: _setIsReferenceLinesEnabled,
-  setIsCrosshairsEnabled: _setIsCrosshairsEnabled,
```
Comment (ryan00012) [inline platform/core/src/services/ViewerToolsetService/ViewerToolset.js:7]: Why is setIsCrosshairsEnabled removed now? Should it have both Crosshairs and seriesLinking?

File: platform/core/src/services/ViewerToolsetService/ViewerToolset.js [hunk 2]
```diff
@@ -4,16 +4,16 @@ const publicAPI = {
...
   setServiceImplementation,
 };
 
 const serviceImplementation = {
   _getState: () => console.warn('getState() NOT IMPLEMENTED'),
   _setIsReferenceLinesEnabled: () =>
     console.warn('setIsReferenceLinesEnabled() NOT IMPLEMENTED'),
-  _setIsCrosshairsEnabled: () =>
-    console.warn('setIsCrosshairsEnabled() NOT IMPLEMENTED'),
+  _setIsSeriesLinkingEnabled: () =>
```
Comment (ryan00012) [inline platform/core/src/services/ViewerToolsetService/ViewerToolset.js:15]: Same here:  Why is setIsCrosshairsEnabled removed now and replaced with series linking. Applies to rest of th e PR too

File: extensions/cornerstone/src/commandsModule.js [hunk 3]
```diff
@@ -150,6 +150,13 @@ const commandsModule = ({ servicesManager, commandsManager }) => {
         cornerstone.updateImage(enabledElement.element);
       });
     },
+    toggleSeriesLinking: () => {
+      const { isSeriesLinkingEnabled } = ViewerToolsetService.getState();
+      ViewerToolsetService.setIsSeriesLinkingEnabled(!isSeriesLinkingEnabled);
+      // cornerstone.getEnabledElements().forEach(enabledElement => {
+      //   cornerstone.updateImage(enabledElement.element);
+      // });
```
Comment (shivasuri) [inline extensions/cornerstone/src/commandsModule.js:158]: Can we delete these lines?

**Review by shivasuri — CHANGES_REQUESTED** [review]
Just tested out locally, and wow it looks great 💯🙌🏻
Just one thing: Series Linking should work for series of same orientation across different studies. Other than that, everything is awesome.

File: extensions/cornerstone/src/commandsModule.js (hunk 3 above)
Comment (legend620) [reply extensions/cornerstone/src/commandsModule.js:158]: yes

File: extensions/cornerstone/src/initSeriesLinking.js [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        ) {
+          return;
+        }
+
+        // if (
+        //   sourceImagePlane.frameOfReferenceUID !==
+        //   targetImagePlane.frameOfReferenceUID
+        // ) {
+        //   return;
+        // }
```
Comment (shivasuri) [inline extensions/cornerstone/src/initSeriesLinking.js:65]: delete?

**Review by shivasuri — APPROVED** [review]

File: extensions/cornerstone/src/initSeriesLinking.js (hunk 4 above)
Comment (legend620) [reply extensions/cornerstone/src/initSeriesLinking.js:65]: yes we can delete

**Pull request merged**